coala_quickstart/generation/Settings.py

```diff
@@ -0,0 +1,105 @@
+import os
+from collections import OrderedDict
+
+from pyprint.NullPrinter import NullPrinter
+from coalib.output.printers.LogPrinter import LogPrinter
+from coala_utils.Extensions import exts
+from coala_quickstart.generation.Utilities import (
+    split_by_language, get_extensions, get_gitignore_glob)
+from coalib.settings.Section import Section
+from coalib.output.ConfWriter import ConfWriter
+from coalib.collecting.Collectors import collect_files
+
+
+def generate_section(section_name, extensions_used, bears):
+    """
+    Generates a section for a particular language (or default).
+
+    :param section_name:
+        Name of the section.
+    :param extensions_used:
+        A list of extensions associated with this section.
+    :param bears:
+        A list of bear classes.
+    :return:
+        A ``Section`` object containing the section.
+    """
+    section = Section(section_name, None)
+
+    section["bears"] = ", ".join(bear.name for bear in bears)
+    section["files"] = ", ".join("**" + ext for ext in extensions_used)
+
+    return section
+
+
+def generate_ignore_field(project_dir, languages, extset, ignore_globs):
+    """
+    Generate the ignore field for the ``default`` section.
+
+    :param project_dir:
+        Full path of the user's project directory.
+    :param languages:
+        A list of languages present in the project.
+    :param extset:
+        A dict with language name as key and a set of extensions as
+        value. This includes only those extensions used by the project.
+    :return:
+        A comma-separated string containing the globs to ignore.
+    """
+    null_printer = LogPrinter(NullPrinter())
+
+    all_files = set(collect_files(
+        "**",
+        null_printer,
+        ignored_file_paths=ignore_globs))
+
+    ignores = [".git/**"]
```

Comment:
this is already done at https://github.com/coala-analyzer/coala-quickstart/pull/24/commits/839fa195102d0d40a185738f23c718127cd41c7d#diff-7e6eaad4d79d2cfe0c15b00a0c204498R44 right?